src/libraries/System.Net.Mail/tests/Functional/System.Net.Mail.Functional.Tests.csproj

```diff
@@ -24,6 +25,8 @@
              Link="Common\System\Diagnostics\Tracing\TestEventListener.cs" />
     <Compile Include="$(CommonTestPath)System\IO\TempFile.cs"
              Link="Common\System\IO\TempFile.cs" />
+    <Compile Include="$(CommonTestPath)System\Net\QuicLoad.cs"
```

Comment:
Did we actually see "Error while reaping child" error in the System.Net.Mail tests or is this just a pre-emptive change?

Reply:
no we did not. There are two failures I'm investigating so it is pre-emptive.